source/docs/v3/tenant-attribute-encryption.blade.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Tenant attribute encryption
+extends: _layouts.documentation
+section: content
+---
+
+# Tenant attribute encryption {#encrypt}
+
+To encrypt the attributes, you must store them in [custom columns](https://tenancyforlaravel.com/docs/v3/tenants/#custom-columns). For example, we'll encrypt the tenant's database credentials (which are stored using the [Virtual Column](https://github.com/archtechx/virtualcolumn) as `tenancy_db_username` and `tenancy_db_password` by default) in a few steps.
```

Comment:
To encrypt **the** attributes doesn't make sense here

(fyi all feedback is for @lukinovec)

Reply:
I don't understand "you must store them in custom columns. For example, we'll encrypt [...] which are stored in the virtual column"

Reply:
Also, the example should come later in the article. The first part should be about what to do, and showing it quickly. A step-by-step thing should be at the end. It's **primarily** documentation, not a tutorial.

Reply:
> To encrypt the attributes doesn't make sense here

It made sense, considering we're on a page about tenant attributes. But 'the tenant attributes' instead of 'the attributes' would make more sense.

> I don't understand "you must store them in custom columns. For example, we'll encrypt [...] which are stored in the virtual column"

"in the virtual column ... by default", implying we'll make the attributes custom. I see that can be unclear.

> The first part should be about what to do, and showing it quickly. A step-by-step thing should be at the end. It's primarily documentation, not a tutorial.

I see. I'll rewrite this.

Reply:
https://github.com/stancl/tenancy-docs/pull/174/commits/e05e86793e8f307f0b875fe769d15db78ed6e91c
Updated the page

Reply:
> It made sense, considering we're on a page about tenant attributes

My point is that it's the first sentence so it cannot refer to "the" attributes. It should explain what attributes first.

Reply:
Now it says "the tenant attributes" which still doesn't make sense. It might not even be clear to the reader what "tenant attributes" means.

It should be something like
> To encrypt attributes on the Tenant model, ...

And **then** you can say "the attributes". But until then, there are no "the" attributes.

Reply:
Right, I understand, I'll change it

Reply:
The page looks good now. Only thing I'd change is `Add custom columns to the tenants migration` — "tenants migration" isn't really a good term.

Also, the page should definitely be moved in the docs. It should be in digging deeper.

Now it's in Concepts, which makes people learn about "Tenant attribute encryption" before even knowing about routes or the event system:
<img width="315" alt="CleanShot 2022-07-20 at 12 58 27@2x" src="https://user-images.githubusercontent.com/33033094/179966220-c36fcc5a-b525-4216-a0e8-e6cb5e3c8f4d.png">

Reply:
> "tenants migration" isn't really a good term.

Maybe "Add custom columns to the `tenants` table" ?

> Also, the page should definitely be moved in the docs. It should be in digging deeper.

Right, makes sense. I'll move it

Reply:
https://github.com/stancl/tenancy-docs/pull/174/commits/d52192ef7e21ed9c315f031886f64eb6e62b7676

We can resolve this if the positioning is fine now